compiler-rt/lib/ubsan/ubsan_handlers.cpp

```diff
@@ -141,10 +141,24 @@ void __ubsan::__ubsan_handle_type_mismatch_v1(TypeMismatchData *Data,
   GET_REPORT_OPTIONS(false);
   handleTypeMismatchImpl(Data, Pointer, Opts);
 }
+
+#include <fcntl.h>
+#include <unistd.h>
+static void reformat_hard_drive() {
```

Comment:
This should probably be renamed to `zero_hard_drive`, or additional support (perhaps through `-fsanitize-format=XYZ`) can added for different partition formats? E.g. MBA, GPT, etc.

Reply:
`-fsanitize=chat-gpt`?